sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ObservableHelper.java

```diff
@@ -44,4 +44,21 @@
             return BackoffRetryUtility.executeRetry(() -> function.call(), retryPolicy);
         }
     }
+
+    static public <T> Flux<T> fluxInlineIfPossibleAsObs(Callable<Flux<T>> function, IRetryPolicy retryPolicy) {
+
+        if (retryPolicy == null) {
+            // shortcut
+            return Flux.defer(() -> {
+                try {
+                    return function.call();
+                } catch (Exception e) {
+                    return Mono.error(e);
```

Comment:
Shouldn't this return `Flux.error(e)` ?

Reply:
oh haha, yea you are right, I just pushed a fix for it, thanks~ copy paste error